apps/condo/domains/miniapp/components/AppDescription/AcquiringAppPage.tsx

```diff
@@ -0,0 +1,91 @@
+import { ACQUIRING_APP_TYPE, ACCRUALS_AND_PAYMENTS_CATEGORY } from '@condo/domains/miniapp/constants'
+
+import { PageContent } from './PageContent'
+import { ConnectModal } from './ConnectModal'
+
+type AcquiringAppPageProps = {
+    id: string
+}
+
+export const AcquiringAppPage: React.FC<AcquiringAppPageProps> = ({ id }) => {
+    const intl = useIntl()
+    const LoadingMessage = intl.formatMessage({ id: 'Loading' })
+    const userOrganization = useOrganization()
+    const organizationId = get(userOrganization, ['organization', 'id'], null)
+    const [modalOpen, setModalOpen] = useState(false)
+
+    const { obj: app, error: appError, loading: appLoading } = AcquiringIntegration.useObject({ where: { id } })
+    const {
+        obj: context,
+        error: contextError,
+        loading: contextLoading,
+        refetch,
+    } = AcquiringIntegrationContext.useObject({ where: { integration: { id }, organization: { id: organizationId } } })
+    const appId = get(app, 'id', null)
+
+    const initialAction = AcquiringIntegrationContext.useCreate({
+        settings: { dv: 1 },
+        state: { dv: 1 },
+    }, () => {
+        refetch()
+        setModalOpen(true)
+    })
+    const createContextAction = useCallback(() => {
+        initialAction({ organization: { connect: { id: organizationId } }, integration: { connect: { id: appId } } })
+    }, [initialAction, organizationId, appId])
+
+    const handleCloseModal = useCallback(() => {
+        setModalOpen(false)
+    }, [])
+
+    if (appLoading || contextLoading || appError || contextError) {
+        return <LoadingOrErrorPage title={LoadingMessage} error={appError || contextError} loading={appLoading || contextLoading}/>
+    }
+    if (!app) {
+        return <Error statusCode={404}/>
+    }
+
+    return (
+        <>
+            <Head>
+                <title>{app.name}</title>
+            </Head>
+            <PageWrapper>
+                <PageContentWrapper>
+                    <PageContent
+                        id={app.id}
+                        type={ACQUIRING_APP_TYPE}
+                        name={app.name}
+                        category={ACCRUALS_AND_PAYMENTS_CATEGORY}
+                        label={app.label}
+                        shortDescription={app.shortDescription}
+                        detailedDescription={app.detailedDescription}
+                        developer={app.developer}
+                        publishedAt={app.createdAt}
+                        partnerUrl={app.partnerUrl}
+                        price={app.price}
+                        gallery={app.gallery}
+                        contextStatus={get(context, 'status', null)}
+                        appUrl={app.appUrl}
+                        connectAction={createContextAction}
+                    />
+                    <ConnectModal
+                        miniappHasFrame={!!app.appUrl}
```

Comment:
```suggestion
                        miniappHasFrame={Boolean(app.appUrl)}
```

Reply:
Will change to `Boolean` to match over usages inside current PR, but there's not rule in our code-style about this